core/templates/components/question-directives/question-player/question-player.directive.ts

```diff
@@ -191,9 +191,9 @@ angular.module('oppia').directive('questionPlayer', [
             if (actionButtonType === 'BOOST_SCORE') {
               iconHtml = `<picture>
               <source type="image/webp" 
-              src="${getStaticImageUrl('/icons/rocket@2x.webp')}">
+              srcset="${getStaticImageUrl('/icons/rocket@2x.webp')}">
               <source type="image/png" 
-              src="${getStaticImageUrl('/icons/rocket@2x.png')}">
+              srcset="${getStaticImageUrl('/icons/rocket@2x.png')}">
               <img class="action-button-icon" src="
               ${getStaticImageUrl('/icons/rocket@2x.png')}"/>
```

Comment:
Just curious, why do we need two elements referring to the same png image? If it is a fallback, isn't it sufficient to keep just the img tag?

Reply:
It is needed in cases when `<picture>` is not supported by the browser, it will just evaluate the `<img>` and skip the rest, on other hand, when `<picture>` is supported but WebP isn't it will use the `<source>` to get the image.

Reply:
Ah got it. Thanks for explaining @vojtechjelinek!